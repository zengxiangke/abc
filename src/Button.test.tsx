import { expect, test } from '@jest/globals'
import Button from './Button'
// @ts-ignore
import renderer from 'react-test-renderer'

test('Button is a thing', () => {
  expect(Button).toBeTruthy()
})

test('Button snapshot', () => {
  const component = renderer.create(<Button />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
