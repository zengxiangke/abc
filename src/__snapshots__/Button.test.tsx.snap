// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button snapshot 1`] = `
<button>
  Click Me
</button>
`;
